src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/full.php

```diff
@@ -74,10 +74,7 @@
             ],
         ],
     ],
-    'property_info' => [
-        'enabled' => true,
-        'with_constructor_extractor' => true,
-    ],
+    'property_info' => true,
```

Comment:
Changes like this were added in #50334 to avoid triggering the deprecation. No need for them any more.